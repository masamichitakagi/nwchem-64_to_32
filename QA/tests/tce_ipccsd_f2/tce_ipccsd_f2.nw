# IPCCSD cc-pVDZ 15.10 eV

start  tce_ipccsd_f2

title "tce_ipccsd_f2"

echo

memory stack 1000 mb heap 200 mb global 500 mb

#permanent_dir /work/kiran/F2/tce_ipccsd_f2/

geometry units angstroms
symmetry c1
 F    0.0000000000        0.0000000000   0.7059650
 F    0.0000000000        0.0000000000  -0.7059650
end

basis spherical
 * library cc-pvdz
end

scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
ipccsd
nroots 1
freeze atomic
#symmetry
#targetsym a1
thresh 1.0e-7
end
#set tce:maxeorb 0.1
task tce energy


