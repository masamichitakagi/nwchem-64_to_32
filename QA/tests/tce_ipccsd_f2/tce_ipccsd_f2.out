 argument  1 = /home/edo/park/nwchem-travisci_test/QA/tests/tce_ipccsd_f2/tce_ipccsd_f2.nw



============================== echo of input deck ==============================
# IPCCSD cc-pVDZ 15.10 eV

start  tce_ipccsd_f2

title "tce_ipccsd_f2"

echo

memory stack 1000 mb heap 200 mb global 500 mb

#permanent_dir /work/kiran/F2/tce_ipccsd_f2/

geometry units angstroms
symmetry c1
 F    0.0000000000        0.0000000000   0.7059650
 F    0.0000000000        0.0000000000  -0.7059650
end

basis spherical
 * library cc-pvdz
end

scf
thresh 1.0e-10
tol2e 1.0e-10
singlet
rhf
end

tce
ipccsd
nroots 1
freeze atomic
#symmetry
#targetsym a1
thresh 1.0e-7
end
#set tce:maxeorb 0.1
task tce energy


================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.8
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = lagrange
    program         = /home/edo/park/nwchem-travisci_test/bin/LINUX64/nwchem
    date            = Sun Dec  3 19:38:17 2017

    compiled        = Sun_Dec_03_19:38:10_2017
    source          = /home/edo/park/nwchem-travisci_test
    nwchem branch   = Development
    nwchem revision = nwchem_on_git-89-gebe7d059dcda6593a67eb6acac074d4248b36a5b
    ga revision     = nwchem_on_git-46-g9980ca6
    use scalapack   = F
    input           = /home/edo/park/nwchem-travisci_test/QA/tests/tce_ipccsd_f2/tce_ipccsd_f2.nw
    prefix          = tce_ipccsd_f2.
    data base       = ./tce_ipccsd_f2.db
    status          = startup
    nproc           =        3
    time left       =     -1s



           Memory information
           ------------------

    heap     =   26214396 doubles =    200.0 Mbytes
    stack    =  131072001 doubles =   1000.0 Mbytes
    global   =   65536000 doubles =    500.0 Mbytes (distinct from heap & stack)
    total    =  222822397 doubles =   1700.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                                   tce_ipccsd_f2
                                   -------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------
     1 autoz failed with cvr_scaling = 1.2 changing to 1.3
  no constraints, skipping    0.0000000000000000     
  no constraints, skipping    0.0000000000000000     


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 F                    9.0000     0.00000000     0.00000000     0.70596500
    2 F                    9.0000     0.00000000     0.00000000    -0.70596500

      Atomic Mass 
      ----------- 

      F                 18.998400


 Effective nuclear repulsion energy (a.u.)      30.3579902467

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.41193


            XYZ format geometry
            -------------------
     2
 geometry
 F                     0.00000000     0.00000000     0.70596500
 F                     0.00000000     0.00000000    -0.70596500



 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pvdz                   on all atoms 


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  F (Fluorine)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.47100000E+04  0.000721
  1 S  2.20700000E+03  0.005553
  1 S  5.02800000E+02  0.028267
  1 S  1.42600000E+02  0.106444
  1 S  4.64700000E+01  0.286814
  1 S  1.67000000E+01  0.448641
  1 S  6.35600000E+00  0.264761
  1 S  1.31600000E+00  0.015333

  2 S  1.47100000E+04 -0.000165
  2 S  2.20700000E+03 -0.001308
  2 S  5.02800000E+02 -0.006495
  2 S  1.42600000E+02 -0.026691
  2 S  4.64700000E+01 -0.073690
  2 S  1.67000000E+01 -0.170776
  2 S  6.35600000E+00 -0.112327
  2 S  1.31600000E+00  0.562814

  3 S  3.89700000E-01  1.000000

  4 P  2.26700000E+01  0.044878
  4 P  4.97700000E+00  0.235718
  4 P  1.34700000E+00  0.508521

  5 P  3.47100000E-01  1.000000

  6 D  1.64000000E+00  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 F                          cc-pvdz                  6       14   3s2p1d


                                 NWChem SCF Module
                                 -----------------


                                   tce_ipccsd_f2



  ao basis        = "ao basis"
  functions       =    28
  atoms           =     2
  closed shells   =     9
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./tce_ipccsd_f2.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 F                          cc-pvdz                  6       14   3s2p1d



 Forming initial guess at       0.8s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -198.76773237

      Non-variational initial energy
      ------------------------------

 Total energy =    -198.493185
 1-e energy   =    -339.232096
 2-e energy   =     110.380921
 HOMO         =      -0.482510
 LUMO         =      -0.177734


 Starting SCF solution at       1.0s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-10
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-10
 ----------------------------------------------


 #quartets = 3.081D+03 #integrals = 2.185D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_ipccsd_f2.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  42413
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -198.6636108525  7.47D-01  3.68D-01      0.6
                 2     -198.6838208800  2.30D-01  1.10D-01      0.6
                 3     -198.6848734418  1.59D-01  8.51D-02      0.6
                 4     -198.6856693524  4.39D-03  1.74D-03      0.8
                 5     -198.6856703758  3.02D-06  2.02D-06      0.9
                 6     -198.6856703758  4.81D-12  2.51D-12      1.3


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -198.685670375763
      One-electron energy =   -338.456324141950
      Two-electron energy =    109.412663519520
 Nuclear repulsion energy =     30.357990246666

        Time for solution =      0.8s


             Final eigenvalues
             -----------------

              1      
    1  -26.4329
    2  -26.4327
    3   -1.7679
    4   -1.4995
    5   -0.8066
    6   -0.8066
    7   -0.7442
    8   -0.6625
    9   -0.6625
   10    0.0963
   11    1.3091
   12    1.3091
   13    1.3522
   14    1.4969
   15    1.4969
   16    1.5754
   17    1.8220
   18    2.3190
   19    3.4772

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    2  Occ=2.000000D+00  E=-2.643266D+01
              MO Center=  2.3D-18, -1.9D-18, -9.4D-11, r^2= 5.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.707652  1 F  s                 15     -0.707652  2 F  s          

 Vector    3  Occ=2.000000D+00  E=-1.767902D+00
              MO Center=  2.9D-18,  3.6D-17,  1.0D-14, r^2= 6.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.354487  1 F  s                 17      0.354487  2 F  s          
     2      0.342196  1 F  s                 16      0.342196  2 F  s          

 Vector    4  Occ=2.000000D+00  E=-1.499494D+00
              MO Center=  4.3D-18,  2.4D-17, -9.0D-15, r^2= 9.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.430585  1 F  s                 17     -0.430585  2 F  s          
     2      0.375473  1 F  s                 16     -0.375473  2 F  s          

 Vector    5  Occ=2.000000D+00  E=-8.066318D-01
              MO Center=  3.0D-17,  4.5D-18,  2.2D-14, r^2= 8.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.470663  1 F  py                19      0.470663  2 F  py         
     8      0.300619  1 F  py                22      0.300619  2 F  py         

 Vector    6  Occ=2.000000D+00  E=-8.066318D-01
              MO Center= -9.9D-17,  3.5D-17,  3.1D-14, r^2= 8.9D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.470663  1 F  px                18      0.470663  2 F  px         
     7      0.300619  1 F  px                21      0.300619  2 F  px         

 Vector    7  Occ=2.000000D+00  E=-7.441822D-01
              MO Center= -4.0D-17,  1.5D-17, -1.1D-14, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.444276  1 F  pz                20     -0.444276  2 F  pz         
     9      0.294008  1 F  pz                23     -0.294008  2 F  pz         

 Vector    8  Occ=2.000000D+00  E=-6.625475D-01
              MO Center=  2.0D-17, -6.2D-17, -2.3D-14, r^2= 9.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.515133  1 F  py                19     -0.515133  2 F  py         
     8      0.345612  1 F  py                22     -0.345612  2 F  py         

 Vector    9  Occ=2.000000D+00  E=-6.625475D-01
              MO Center=  4.0D-17, -1.0D-17, -3.1D-14, r^2= 9.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.515133  1 F  px                18     -0.515133  2 F  px         
     7      0.345612  1 F  px                21     -0.345612  2 F  px         

 Vector   10  Occ=0.000000D+00  E= 9.627397D-02
              MO Center=  1.6D-18,  5.2D-18,  7.9D-15, r^2= 1.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.562785  1 F  pz                23      0.562785  2 F  pz         
     6      0.503558  1 F  pz                20      0.503558  2 F  pz         
     3     -0.280196  1 F  s                 17      0.280196  2 F  s          

 Vector   11  Occ=0.000000D+00  E= 1.309118D+00
              MO Center=  6.0D-16,  2.2D-16, -2.4D-14, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.617115  1 F  px                18      0.617115  2 F  px         
     7     -0.606321  1 F  px                21     -0.606321  2 F  px         
     5      0.224088  1 F  py                19      0.224088  2 F  py         
     8     -0.220169  1 F  py                22     -0.220169  2 F  py         

 Vector   12  Occ=0.000000D+00  E= 1.309118D+00
              MO Center= -5.7D-17,  1.6D-16, -2.1D-14, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.617115  1 F  py                19      0.617115  2 F  py         
     8     -0.606321  1 F  py                22     -0.606321  2 F  py         
     4     -0.224088  1 F  px                18     -0.224088  2 F  px         
     7      0.220169  1 F  px                21      0.220169  2 F  px         

 Vector   13  Occ=0.000000D+00  E= 1.352174D+00
              MO Center=  4.1D-17, -1.2D-17,  4.6D-14, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.106040  1 F  pz                23      1.106040  2 F  pz         
     6     -0.610365  1 F  pz                20     -0.610365  2 F  pz         
     2     -0.240774  1 F  s                 16      0.240774  2 F  s          
     3     -0.218098  1 F  s                 17      0.218098  2 F  s          

 Vector   14  Occ=0.000000D+00  E= 1.496881D+00
              MO Center=  6.1D-18,  8.3D-18,  2.5D-14, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.793467  1 F  px                21     -0.793467  2 F  px         
     4     -0.559279  1 F  px                18      0.559279  2 F  px         
     8     -0.432763  1 F  py                22      0.432763  2 F  py         
     5      0.305035  1 F  py                19     -0.305035  2 F  py         

 Vector   15  Occ=0.000000D+00  E= 1.496881D+00
              MO Center= -1.0D-17, -3.6D-18,  2.0D-14, r^2= 1.6D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.793467  1 F  py                22     -0.793467  2 F  py         
     5     -0.559279  1 F  py                19      0.559279  2 F  py         
     7      0.432763  1 F  px                21     -0.432763  2 F  px         
     4     -0.305035  1 F  px                18      0.305035  2 F  px         

 Vector   16  Occ=0.000000D+00  E= 1.575402D+00
              MO Center= -3.8D-16, -3.4D-16, -4.9D-14, r^2= 2.0D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.723941  1 F  s                 17      0.723941  2 F  s          
     2     -0.640466  1 F  s                 16     -0.640466  2 F  s          
     9      0.594161  1 F  pz                23     -0.594161  2 F  pz         
     6     -0.543932  1 F  pz                20      0.543932  2 F  pz         
     1     -0.257470  1 F  s                 15     -0.257470  2 F  s          

 Vector   17  Occ=0.000000D+00  E= 1.822006D+00
              MO Center= -2.6D-16, -2.0D-16, -2.2D-15, r^2= 7.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.887075  1 F  s                 16      0.887075  2 F  s          
     3     -0.683608  1 F  s                 17     -0.683608  2 F  s          
     6     -0.458011  1 F  pz                20      0.458011  2 F  pz         
     9      0.381400  1 F  pz                23     -0.381400  2 F  pz         
     1      0.349716  1 F  s                 15      0.349716  2 F  s          

 Vector   18  Occ=0.000000D+00  E= 2.318984D+00
              MO Center= -2.4D-17, -7.7D-18,  5.6D-15, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.839311  1 F  s                 17     -1.839311  2 F  s          
     2     -1.215949  1 F  s                 16      1.215949  2 F  s          
     9     -0.917433  1 F  pz                23     -0.917433  2 F  pz         
     1     -0.544665  1 F  s                 15      0.544665  2 F  s          
     6      0.160614  1 F  pz                20      0.160614  2 F  pz         

 Vector   19  Occ=0.000000D+00  E= 3.477191D+00
              MO Center= -7.5D-18, -1.7D-18,  6.8D-15, r^2= 8.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      0.675222  1 F  d  0              26      0.675222  2 F  d  0       


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
          67.625585016783           0.000000000000           0.000000000000
           0.000000000000          67.625585016783           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 F    9     9.00   2.00  0.94  1.03  3.18  1.84  0.01
    2 F    9     9.00   2.00  0.94  1.03  3.18  1.84  0.01

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0     -0.000000      0.000000     18.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0     -0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000

     2   2 0 0     -6.961654      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -6.961654      0.000000      0.000000
     2   0 1 1     -0.000000      0.000000      0.000000
     2   0 0 2     -6.495530      0.000000     32.035870


 Parallel integral file used       3 records with       0 large values

                   NWChem Extensible Many-Electron Theory Module
                   ---------------------------------------------

              ======================================================
                   This portion of the program was automatically
                  generated by a Tensor Contraction Engine (TCE).
                  The development of this portion of the program
                 and TCE was supported by US Department of Energy,
                Office of Science, Office of Basic Energy Science.
                      TCE is a product of Battelle and PNNL.
              Please cite: S.Hirata, J.Phys.Chem.A 107, 9887 (2003).
              ======================================================

                                   tce_ipccsd_f2


            General Information
            -------------------
      Number of processors :     3
         Wavefunction type : Restricted Hartree-Fock
          No. of electrons :    18
           Alpha electrons :     9
            Beta electrons :     9
           No. of orbitals :    56
            Alpha orbitals :    28
             Beta orbitals :    28
        Alpha frozen cores :     2
         Beta frozen cores :     2
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity : singlet 
    Number of AO functions :    28
       Number of AO shells :    12
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-09

          Correlation Information
          -----------------------
          Calculation type : Coupled-cluster singles & doubles                           
   Perturbative correction : none                                                        
            Max iterations :      100
        Residual threshold : 0.10D-06
     T(0) DIIS level shift : 0.00D+00
     L(0) DIIS level shift : 0.00D+00
     T(1) DIIS level shift : 0.00D+00
     L(1) DIIS level shift : 0.00D+00
     T(R) DIIS level shift : 0.00D+00
     T(I) DIIS level shift : 0.00D+00
   CC-T/L Amplitude update :  5-th order DIIS
     No. of excited states :     1
               Target root :     1
           Target symmetry : none
      Symmetry restriction : off
   Dipole & oscillator str : off
                I/O scheme : Global Array Library
        L-threshold :  0.10D-06
        EOM-threshold :  0.10D-06
 no EOMCCSD initial starts read in
 TCE RESTART OPTIONS
 READ_INT:   F
 WRITE_INT:  F
 READ_TA:    F
 WRITE_TA:   F
 READ_XA:    F
 WRITE_XA:   F
 READ_IN3:   F
 WRITE_IN3:  F
 SLICE:      F
 D4D5:       F

            Memory Information
            ------------------
          Available GA space size is     196607216 doubles
          Available MA space size is     157284692 doubles

 Maximum block size        32 doubles

 tile_dim =     19

 Block   Spin    Irrep     Size     Offset   Alpha
 -------------------------------------------------
   1    alpha     a      7 doubles       0       1
   2    beta      a      7 doubles       7       1
   3    alpha     a     19 doubles      14       3
   4    beta      a     19 doubles      33       3

 Global array virtual files algorithm will be used

 Parallel file system coherency ......... OK

 #quartets = 3.081D+03 #integrals = 2.185D+04 #direct =  0.0% #cached =100.0%


 Integral file          = ./tce_ipccsd_f2.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  42413
 No. of bits per label  =      8        No. of bits per value  =     64


File balance: exchanges=     0  moved=     0  time=   0.0


 Fock matrix recomputed
 1-e file size   =              676
 1-e file name   = ./tce_ipccsd_f2.f1
 Cpu & wall time / sec            0.0            0.1

 tce_ao2e: fast2e=1
 half-transformed integrals in memory

 2-e (intermediate) file size =         1381408
 2-e (intermediate) file name = ./tce_ipccsd_f2.v2i
 Cpu & wall time / sec            0.2            0.4

 tce_mo2e: fast2e=1
 2-e integrals stored in memory

 2-e file size   =           751825
 2-e file name   = ./tce_ipccsd_f2.v2
 Cpu & wall time / sec            0.0            0.1
 T1-number-of-tasks                    1

 t1 file size   =              133
 t1 file name   = ./tce_ipccsd_f2.t1
 t1 file handle =       -999
 T2-number-of-boxes                    2

 t2 file size   =            35378
 t2 file name   = ./tce_ipccsd_f2.t2
 t2 file handle =       -996

 CCSD iterations
 -----------------------------------------------------------------
 Iter          Residuum       Correlation     Cpu    Wall    V2*C2
 -----------------------------------------------------------------
    1   0.1703541754138  -0.3939433686074     1.4     3.3     0.0
    2   0.0469817497123  -0.3926753885086     1.4     3.2     0.0
    3   0.0186955315330  -0.4004922181422     0.9     2.4     0.0
    4   0.0089821612376  -0.4013832771646     0.9     2.8     0.0
    5   0.0046067593996  -0.4022228723319     0.6     1.9     0.0
 MICROCYCLE DIIS UPDATE:                    5                    5
    6   0.0015583069428  -0.4027120234565     1.3     3.1     0.0
    7   0.0010296082060  -0.4027696551102     0.9     2.4     0.0
    8   0.0007212005366  -0.4027750110451     1.6     3.9     0.1
    9   0.0005143983369  -0.4027769459386     2.1     4.6     0.1
   10   0.0003675170947  -0.4027802671115     3.1     6.0     0.1
 MICROCYCLE DIIS UPDATE:                   10                    5
   11   0.0000226731816  -0.4027890881001     1.5     3.4     0.1
   12   0.0000095215543  -0.4027877162163     2.4     5.1     0.0
   13   0.0000046112748  -0.4027882400284     1.5     3.9     0.1
   14   0.0000024473860  -0.4027881928271     1.2     2.8     0.0
   15   0.0000013539915  -0.4027882062418     2.5     5.9     0.0
 MICROCYCLE DIIS UPDATE:                   15                    5
   16   0.0000003377790  -0.4027881926617     3.4     6.8     0.0
   17   0.0000002174648  -0.4027881893815     1.9     4.2     0.1
   18   0.0000001481297  -0.4027881945010     0.8     2.0     0.0
   19   0.0000001052287  -0.4027881940212     2.4     5.1     0.1
   20   0.0000000749117  -0.4027881950021     1.4     3.1     0.1
 -----------------------------------------------------------------
 Iterations converged
 CCSD correlation energy / hartree =        -0.402788195002103
 CCSD total energy / hartree       =      -199.088458570765312

 Singles contributions

 Doubles contributions
    10a   (alpha)    10a   (beta ) ---     7a   (alpha)     7a   (beta )       -0.1856935044

 IPCCSD calculation

 Ground-state symmetry is a   

 =========================================
 Excited-state calculation ( a   symmetry)
 =========================================
 Dim. of EOMCC iter. space      500

 x1 file size   =                7

 x2 file size   =             1862

 EOMCCSD SOLVER TYPE  1

 No. of initial right vectors    2

 EOM-CCSD right-hand side iterations
 --------------------------------------------------------------
      Residuum       Omega / hartree  Omega / eV    Cpu    Wall
 --------------------------------------------------------------

 Iteration   1 using    2 trial vectors
   0.6672841420594   0.7199115305086   19.58980     1.5     3.2

 Iteration   2 using    3 trial vectors
   0.0912121312899   0.5611252024747   15.26900     0.5     1.2

 Iteration   3 using    4 trial vectors
   0.0216655390842   0.5554934727611   15.11575     0.9     1.8

 Iteration   4 using    5 trial vectors
   0.0070510173410   0.5549733866786   15.10160     0.3     1.1

 Iteration   5 using    6 trial vectors
   0.0022285557718   0.5549282612674   15.10037     0.8     1.5

 Iteration   6 using    7 trial vectors
   0.0005553344462   0.5548873710063   15.09926     0.6     1.2

 Iteration   7 using    8 trial vectors
   0.0001197113703   0.5548805673550   15.09907     1.8     4.0

 Iteration   8 using    9 trial vectors
   0.0000190559104   0.5548804895205   15.09907     1.5     3.0

 Iteration   9 using   10 trial vectors
   0.0000031737488   0.5548804951819   15.09907     1.2     2.3

 Iteration  10 using   11 trial vectors
   0.0000005048535   0.5548805139595   15.09907     0.6     1.4

 Iteration  11 using   12 trial vectors
   0.0000000956882   0.5548805187113   15.09907     1.1     2.0
 --------------------------------------------------------------
 Iterations converged
 ------------------

 Excited state root  1
 Excitation energy / hartree =        0.554880518711318
                   / eV      =       15.099073582827138
 largest EOMCCSD amplitudes: R1 and R2

 Singles contributions
     9a   (alpha)        0.9546001246

 Doubles contributions
    10a   (alpha)  ---     6a   (alpha)     7a   (alpha)       -0.1099399166
    10a   (alpha)  ---     7a   (alpha)     6a   (alpha)        0.1099399166
    10a   (beta )  ---     6a   (alpha)     7a   (beta )       -0.1500096629

 Parallel integral file used       3 records with       0 large values


 Task  times  cpu:       46.0s     wall:      102.6s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls: 4259     4259     1.07e+04 4585     4431        0        0     7514     
number of processes/call 1.61e+00 1.11e+00 1.69e+00 0.00e+00 0.00e+00
bytes total:             5.38e+08 3.15e+07 1.29e+08 0.00e+00 0.00e+00 6.01e+04
bytes remote:            1.85e+08 1.58e+07 4.85e+07 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 5696720 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        30
	current total bytes		         0	         0
	maximum total bytes		   1060896	  22509608
	maximum total K-bytes		      1061	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
    S. A. Fischer, S. Krishnamoorthy, M. Jacquelin, W. Ma, M. Klemm, O. Villa,
      Y. Chen, V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov,
      T. Risthaus, M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin,
        P. Nichols, R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari,
       M. Williamson, R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:       46.1s     wall:      102.8s
